src/Support/Facade.php

```diff
@@ -19,6 +19,7 @@ protected static function resolveFacadeInstance($name)
     {
         if (
             !is_object($name) &&
+            !is_null(static::$app) &&
```

Comment:
Why was this required?

Reply:
@LukeTowers some unit tests fail without this, as no application is available (I believe only the plugin test cases initiate an application).